src/components/RevokeContainer/RevokeDocumentTileArea/RevokeDocumentTileArea.tsx

```diff
@@ -36,6 +36,7 @@ export const RevokeDocumentTileArea: FunctionComponent<RevokeDocumentTileArea> =
           <ProgressBar step={2} totalSteps={3} />
           <h3 className="my-8 pb-8 border-b">Confirm Document</h3>
           <ProcessedDocumentTag
+            hideAction
             doc={revokeDocuments[0]}
```

Comment:
`hideAction` if its need or no in this case?

Reply:
this one is shorthand for hideAction={true} which means hide the action